CMakeLists.txt

```diff
@@ -785,6 +785,8 @@ if(openPMD_HAVE_ADIOS1)
 
     target_link_libraries(openPMD PUBLIC openPMD.ADIOS1.Serial)
     target_link_libraries(openPMD PUBLIC openPMD.ADIOS1.Parallel)
+else()
+    target_sources(openPMD PRIVATE ${IO_ADIOS1_SEQUENTIAL_SOURCE} ${IO_ADIOS1_SOURCE})
```

Comment:
```suggestion
    # add stubs to prevent missing symbols in Clang ASAN/UBSAN
    target_sources(openPMD PRIVATE ${IO_ADIOS1_SEQUENTIAL_SOURCE} ${IO_ADIOS1_SOURCE})
```